### PR TITLE
Add Monokai Pro Theme and Fix Checkboxes in PR Template

### PR DESCRIPTION
**Pull request by sreetamdas**
## Name of an extension you are adding

Monokai Pro

## Why do you think this extension is awesome?

Its a beautiful theme, with an optional free license (only popups, no limits to usage)!

## Make sure that:

- [x] Screenshot/GIF included (to demonstrate the plugin functionality)

- [x] ToC updated

### CODE REVIEW

**Comment by sreetamdas** [comment]
Also, fixed the checkboxes in the PULL_REQUEST_TEMPLATE, adding a help text in the comments.

File: README.md [hunk 1]
```diff
@@ -129,10 +129,12 @@ A curated list of delightful [Visual Studio Code](https://code.visualstudio.com/
   - [Material Palenight Theme by whizkydee](#material-palenight-theme-by-whizkydee)
   - [Material Theme by Mattia Astorino](#material-theme-by-mattia-astorino)
   - [Mno by u29dc](#mno-by-u29dc)
+  - [Monokai Pro by monokai](#monokai-pro-by-monokai)
   - [Night Owl by Sarah Drasner](#night-owl-by-sarah-drasner)
   - [Plastic by Will Stone](#plastic-by-will-stone)
   - [Nord by arcticicestudio](#nord-by-arcticicestudio)
   - [Shades of Purple by Ahmad Awais](#shades-of-purple-by-ahmad-awais)
+  - [Monokai Pro (commercial)](#monokai-pro)
```
[the remark this answers is not in the document]
Comment (sreetamdas) [reply README.md]: My bad, I forgot to remove the duplicate entry; thanks for pointing it out 😄

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -8,6 +8,10 @@
...
 
-[ ] Screenshot/GIF included (to demonstrate the plugin functionality)
+<!-- 
+Check off the checkboxes with an 'x' like this: [x] 
+-->
 
-[ ] ToC updated
+- [ ] Screenshot/GIF included (to demonstrate the plugin functionality)
+
+- [ ] ToC updated
```
Comment (jschr) [inline PULL_REQUEST_TEMPLATE.md]: @viatsko You okay with these changes?

**Comment by viatsko** [comment]
"Commercial" badge should be added in both places

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (viatsko) [reply PULL_REQUEST_TEMPLATE.md]: LGTM

**Comment by jschr** [comment]
@sreetamdas I think you need to update the #href as well. The ToC link isn't working: https://github.com/viatsko/awesome-vscode/blob/0ba784e6ddca9c9e1e47ec8be98d69973eda3d22/README.md

**Comment by jschr** [comment]
Thanks! 👍

**Pull request merged**